### PR TITLE
chore: bump mods that have GA clients to v1

### PR DESCRIPTION
**Pull request by codyoss**
This commit will be tagged:
- accessapproval/v1.0.0
- aiplatform/v1.0.0
- apigateway/v1.0.0
- apigeeconnect/v1.0.0
- appengine/v1.0.0
- artifactregistry/v1.0.0
- asset/v1.0.0
- automl/v1.0.0
- billing/v1.0.0
- channel/v1.0.0
- cloudbuild/v1.0.0
- clouddms/v1.0.0
- cloudtasks/v1.0.0
- container/v1.0.0
- datacatalog/v1.0.0
- datafusion/v1.0.0
- dataproc/v1.0.0
- dialogflow/v1.0.0
- dlp/v1.0.0
- documentai/v1.0.0
- essentialcontacts/v1.0.0
- eventarc/v1.0.0
- functions/v1.0.0
- gaming/v1.0.0
- gsuiteaddons/v1.0.0
- iap/v1.0.0
- iot/v1.0.0
- kms/v1.0.0
- language/v1.0.0
- managedidentities/v1.0.0
- memcache/v1.0.0
- metastore/v1.0.0
- monitoring/v1.0.0
- networkmanagement/v1.0.0
- orgpolicy/v1.0.0
- osconfig/v1.0.0
- oslogin/v1.0.0
- policytroubleshooter/v1.0.0
- recaptchaenterprise/v1.0.0
- recommender/v1.0.0
- redis/v1.0.0
- resourcemanager/v1.0.0
- resourcesettings/v1.0.0
- retail/v1.0.0
- scheduler/v1.0.0
- secretmanager/v1.0.0
- security/v1.0.0
- securitycenter/v1.0.0
- servicecontrol/v1.0.0
- servicedirectory/v1.0.0
- servicemanagement/v1.0.0
- serviceusage/v1.0.0
- shell/v1.0.0
- speech/v1.0.0
- storagetransfer/v1.0.0
- texttospeech/v1.0.0
- tpu/v1.0.0
- trace/v1.0.0
- translate/v1.0.0
- video/v1.0.0
- videointelligence/v1.0.0
- vision/v1.0.0
- vpcaccess/v1.0.0
- webrisk/v1.0.0
- websecurityscanner/v1.0.0
- workflows/v1.0.0

### CODE REVIEW

**Comment by codyoss** [comment]
Commands that should be run post merge:

```
git tag accessapproval/v1.0.0 $COMMIT_SHA
git tag aiplatform/v1.0.0 $COMMIT_SHA
git tag apigateway/v1.0.0 $COMMIT_SHA
git tag apigeeconnect/v1.0.0 $COMMIT_SHA
git tag appengine/v1.0.0 $COMMIT_SHA
git tag artifactregistry/v1.0.0 $COMMIT_SHA
git tag asset/v1.0.0 $COMMIT_SHA
git tag automl/v1.0.0 $COMMIT_SHA
git tag billing/v1.0.0 $COMMIT_SHA
git tag channel/v1.0.0 $COMMIT_SHA
git tag cloudbuild/v1.0.0 $COMMIT_SHA
git tag clouddms/v1.0.0 $COMMIT_SHA
git tag cloudtasks/v1.0.0 $COMMIT_SHA
git tag container/v1.0.0 $COMMIT_SHA
git tag datacatalog/v1.0.0 $COMMIT_SHA
git tag datafusion/v1.0.0 $COMMIT_SHA
git tag dataproc/v1.0.0 $COMMIT_SHA
git tag dialogflow/v1.0.0 $COMMIT_SHA
git tag dlp/v1.0.0 $COMMIT_SHA
git tag documentai/v1.0.0 $COMMIT_SHA
git tag essentialcontacts/v1.0.0 $COMMIT_SHA
git tag eventarc/v1.0.0 $COMMIT_SHA
git tag functions/v1.0.0 $COMMIT_SHA
git tag gaming/v1.0.0 $COMMIT_SHA
git tag gsuiteaddons/v1.0.0 $COMMIT_SHA
git tag iap/v1.0.0 $COMMIT_SHA
git tag iot/v1.0.0 $COMMIT_SHA
git tag kms/v1.0.0 $COMMIT_SHA
git tag language/v1.0.0 $COMMIT_SHA
git tag managedidentities/v1.0.0 $COMMIT_SHA
git tag memcache/v1.0.0 $COMMIT_SHA
git tag metastore/v1.0.0 $COMMIT_SHA
git tag monitoring/v1.0.0 $COMMIT_SHA
git tag networkmanagement/v1.0.0 $COMMIT_SHA
git tag orgpolicy/v1.0.0 $COMMIT_SHA
git tag osconfig/v1.0.0 $COMMIT_SHA
git tag oslogin/v1.0.0 $COMMIT_SHA
git tag policytroubleshooter/v1.0.0 $COMMIT_SHA
git tag recaptchaenterprise/v1.0.0 $COMMIT_SHA
git tag recommender/v1.0.0 $COMMIT_SHA
git tag redis/v1.0.0 $COMMIT_SHA
git tag resourcemanager/v1.0.0 $COMMIT_SHA
git tag resourcesettings/v1.0.0 $COMMIT_SHA
git tag retail/v1.0.0 $COMMIT_SHA
git tag scheduler/v1.0.0 $COMMIT_SHA
git tag secretmanager/v1.0.0 $COMMIT_SHA
git tag security/v1.0.0 $COMMIT_SHA
git tag securitycenter/v1.0.0 $COMMIT_SHA
git tag servicecontrol/v1.0.0 $COMMIT_SHA
git tag servicedirectory/v1.0.0 $COMMIT_SHA
git tag servicemanagement/v1.0.0 $COMMIT_SHA
git tag serviceusage/v1.0.0 $COMMIT_SHA
git tag shell/v1.0.0 $COMMIT_SHA
git tag speech/v1.0.0 $COMMIT_SHA
git tag storagetransfer/v1.0.0 $COMMIT_SHA
git tag texttospeech/v1.0.0 $COMMIT_SHA
git tag tpu/v1.0.0 $COMMIT_SHA
git tag trace/v1.0.0 $COMMIT_SHA
git tag translate/v1.0.0 $COMMIT_SHA
git tag video/v1.0.0 $COMMIT_SHA
git tag videointelligence/v1.0.0 $COMMIT_SHA
git tag vision/v1.0.0 $COMMIT_SHA
git tag vpcaccess/v1.0.0 $COMMIT_SHA
git tag webrisk/v1.0.0 $COMMIT_SHA
git tag websecurityscanner/v1.0.0 $COMMIT_SHA
git tag workflows/v1.0.0 $COMMIT_SHA
git push $REMOTE refs/tags/accessapproval/v1.0.0
git push $REMOTE refs/tags/aiplatform/v1.0.0
git push $REMOTE refs/tags/apigateway/v1.0.0
git push $REMOTE refs/tags/apigeeconnect/v1.0.0
git push $REMOTE refs/tags/appengine/v1.0.0
git push $REMOTE refs/tags/artifactregistry/v1.0.0
git push $REMOTE refs/tags/asset/v1.0.0
git push $REMOTE refs/tags/automl/v1.0.0
git push $REMOTE refs/tags/billing/v1.0.0
git push $REMOTE refs/tags/channel/v1.0.0
git push $REMOTE refs/tags/cloudbuild/v1.0.0
git push $REMOTE refs/tags/clouddms/v1.0.0
git push $REMOTE refs/tags/cloudtasks/v1.0.0
git push $REMOTE refs/tags/container/v1.0.0
git push $REMOTE refs/tags/datacatalog/v1.0.0
git push $REMOTE refs/tags/datafusion/v1.0.0
git push $REMOTE refs/tags/dataproc/v1.0.0
git push $REMOTE refs/tags/dialogflow/v1.0.0
git push $REMOTE refs/tags/dlp/v1.0.0
git push $REMOTE refs/tags/documentai/v1.0.0
git push $REMOTE refs/tags/essentialcontacts/v1.0.0
git push $REMOTE refs/tags/eventarc/v1.0.0
git push $REMOTE refs/tags/functions/v1.0.0
git push $REMOTE refs/tags/gaming/v1.0.0
git push $REMOTE refs/tags/gsuiteaddons/v1.0.0
git push $REMOTE refs/tags/iap/v1.0.0
git push $REMOTE refs/tags/iot/v1.0.0
git push $REMOTE refs/tags/kms/v1.0.0
git push $REMOTE refs/tags/language/v1.0.0
git push $REMOTE refs/tags/managedidentities/v1.0.0
git push $REMOTE refs/tags/memcache/v1.0.0
git push $REMOTE refs/tags/metastore/v1.0.0
git push $REMOTE refs/tags/monitoring/v1.0.0
git push $REMOTE refs/tags/networkmanagement/v1.0.0
git push $REMOTE refs/tags/orgpolicy/v1.0.0
git push $REMOTE refs/tags/osconfig/v1.0.0
git push $REMOTE refs/tags/oslogin/v1.0.0
git push $REMOTE refs/tags/policytroubleshooter/v1.0.0
git push $REMOTE refs/tags/recaptchaenterprise/v1.0.0
git push $REMOTE refs/tags/recommender/v1.0.0
git push $REMOTE refs/tags/redis/v1.0.0
git push $REMOTE refs/tags/resourcemanager/v1.0.0
git push $REMOTE refs/tags/resourcesettings/v1.0.0
git push $REMOTE refs/tags/retail/v1.0.0
git push $REMOTE refs/tags/scheduler/v1.0.0
git push $REMOTE refs/tags/secretmanager/v1.0.0
git push $REMOTE refs/tags/security/v1.0.0
git push $REMOTE refs/tags/securitycenter/v1.0.0
git push $REMOTE refs/tags/servicecontrol/v1.0.0
git push $REMOTE refs/tags/servicedirectory/v1.0.0
git push $REMOTE refs/tags/servicemanagement/v1.0.0
git push $REMOTE refs/tags/serviceusage/v1.0.0
git push $REMOTE refs/tags/shell/v1.0.0
git push $REMOTE refs/tags/speech/v1.0.0
git push $REMOTE refs/tags/storagetransfer/v1.0.0
git push $REMOTE refs/tags/texttospeech/v1.0.0
git push $REMOTE refs/tags/tpu/v1.0.0
git push $REMOTE refs/tags/trace/v1.0.0
git push $REMOTE refs/tags/translate/v1.0.0
git push $REMOTE refs/tags/video/v1.0.0
git push $REMOTE refs/tags/videointelligence/v1.0.0
git push $REMOTE refs/tags/vision/v1.0.0
git push $REMOTE refs/tags/vpcaccess/v1.0.0
git push $REMOTE refs/tags/webrisk/v1.0.0
git push $REMOTE refs/tags/websecurityscanner/v1.0.0
git push $REMOTE refs/tags/workflows/v1.0.0
```

**Comment by codyoss** [comment]
This commit was generated by running:

```
go run cmd/stabilizer/main.go -parent=/Users/codyoss/oss/google-cloud-go \
-child=accessapproval,aiplatform,apigateway,apigeeconnect,appengine,artifactregistry,asset,automl,billing,channel,cloudbuild,clouddms,cloudtasks,container,datacatalog,datafusion,dataproc,dialogflow,dlp,documentai,essentialcontacts,eventarc,functions,gaming,gsuiteaddons,iap,iot,kms,language,managedidentities,memcache,metastore,monitoring,networkmanagement,orgpolicy,osconfig,oslogin,policytroubleshooter,recaptchaenterprise,recommender,redis,resourcemanager,resourcesettings,retail,scheduler,secretmanager,security,securitycenter,servicecontrol,servicedirectory,servicemanagement,serviceusage,shell,speech,storagetransfer,texttospeech,tpu,trace,translate,video,videointelligence,vision,vpcaccess,webrisk,websecurityscanner,workflows
```

File: accessapproval/CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## 1.0.0
+
+Stabilize GA surfaces.
```
Comment (tbpg) [inline accessapproval/CHANGES.md:5]: Nit: I don't have a better way to phrase it, but it seems funny to have surfaces be plural when this is a single module. :man_shrugging:

**Review by tbpg — APPROVED** [review]

File: accessapproval/CHANGES.md (hunk 1 above)
Comment (codyoss) [reply accessapproval/CHANGES.md:5]: Updated to make singular.

**Pull request merged**